src/components/atoms/Button/Button.tsx

```diff
@@ -54,7 +54,9 @@ export const buttonStyle = css<IButtonProps>`
   ${({ styling = 'primary' }) => {
     const { bg } = colorMap[styling];
     const isActionGradient = bg === colors.action;
-    const bgFallback = `background-color: ${colors.actionPlain}`;
+    const bgFallback = css`
```

Comment:
This was making the button look ugly 😓 Maybe there are some other bugs but we can iterate over it.